Misc/NEWS.d/next/Core and Builtins/2020-08-13-07-19-21.bpo-1653741.fubBkb.rst

```diff
@@ -0,0 +1 @@
+Port :mod:`sha3` to multi-phase init.  Convert static types to heap types.

```

Comment:
Oh... I cannot merge the PR, there is a typo in the module name (_sha3, not sha3):

```suggestion
Port the :mod:`_sha3` extension module to the multi-phase initialization API (:pep:`489`).
Convert also its static types to heap types.
```